### PR TITLE
database: minor fixes for documentation

### PR DESCRIPTION
**Pull request by cataldor**
Hello,
commit 7290f779 suggested mpd_recv_entity() for mpd_send_list_all and mpd_send_list_all_meta; but it didn't for mpd_send_list_all. I imagine that this was an oversight?
Additionally, i've tried to correct some other minor mistakes

### CODE REVIEW

File: include/mpd/database.h [hunk 1]
```diff
@@ -80,14 +82,14 @@ mpd_send_list_all_meta(struct mpd_connection *connection, const char *path);
  * To read the response, you may use mpd_recv_entity().
  *
  * @param connection the connection to MPD
- * @param path the directory to be listed
+ * @param path an optional the directory to be listed
```
Comment (MaxKellermann) [inline include/mpd/database.h]: Typo `the`?

**Comment by MaxKellermann** [comment]
Thanks, but next time please add explantory texts to commit messages, not to PRs to preserve it for the future.

File: include/mpd/database.h (hunk 1 above)
Comment (cataldor) [reply include/mpd/database.h]: yup, i've added a new commit to fix this

**Comment by cataldor** [comment]
Ok.
if you prefer, i can generate a new PR with the explanation

**Comment by MaxKellermann** [comment]
No that's ok this time, just remember that when you do write an explanation (which is good!), consider where you put it. Stuff written only on GitHub is lost, but is available forever in the git history.

**Pull request merged**